ci/docker/runtime_functions.sh

```diff
@@ -897,7 +870,6 @@ build_ubuntu_gpu_cmake_no_rtc() {
         -DUSE_CUDA=ON                           \
         -DUSE_CUDNN=ON                          \
         -DUSE_TVM_OP=ON                         \
```

Comment:
I think we should turn off this by default and create a separate to test TVM_ON instead.